src/components/Button.jsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import styles from "./Button.module.css";
+
+const defaultTheme = {
+  color: "#2b3bcd",
+  background: "white",
+  hover: "#535bf2",
+};
+
+export const Button = ({ children, to = "#", theme = {} }) => {
+  theme = { ...defaultTheme, ...theme };
```

Comment:
[NIT] this was surprising to me and I had to do a double check to see why you were re-assigning the parameter twice. Maybe a const like `themeWithDefaults` would make the code intention clearer. Again: Nit and you can resolve